README.md

```diff
@@ -38,7 +38,7 @@ $ brew install xhyve
 ```
 $ git clone https://github.com/ailispaw/boot2docker-xhyve
 $ cd boot2docker-xhyve
-$ make
+$ make    # You may be asked for your sudo password
```

Comment:
Oh, I see. Yes you’re right. I was thinking it used to say “You may need sudo”. Apologies. I’ll correct it.

On Tue, Oct 6, 2015 at 11:34 AM, A.I. <notifications@github.com> wrote:

>> @@ -38,7 +38,7 @@ $ brew install xhyve
>>  ```
>>  $ git clone https://github.com/ailispaw/boot2docker-xhyve
>>  $ cd boot2docker-xhyve
>> -$ make
>> +$ make    # You may be asked for your sudo password
> Hmm. It is not supposed to need sudo to make only. I will check it.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ailispaw/boot2docker-xhyve/pull/19/files#r41280257